README.md

```diff
@@ -47,17 +47,21 @@ Outside of Bianca, this can be limited to these two or three steps:
 
    For the SDA to be able to understand the encrypted files when they are
    uploaded, they need to be encrypted with the correct public key. This key can
-   be downloaded from this repository with this command:
+   be downloaded from this repository with this command <strong>if you are uploading to the SDA</strong>:
    ```bash
    wget https://raw.githubusercontent.com/NBISweden/EGA-SE-user-docs/main/crypt4gh_key.pub
    ```
+   or this command <strong>if you are uploading to Big Picture</strong>:
+   ```bash
+   wget https://raw.githubusercontent.com/NBISweden/EGA-SE-user-docs/main/crypt4gh_bp_key.pub
```

Comment:
download didn't work (while the one for SDA worked)

Reply:
I think that's because the PR is not merged, I will check it once that's done